assets/js/modules/analytics-4/components/dashboard/AudienceSegmentation/AudienceSegmentationIntroductoryOverlayNotification.stories.js

```diff
@@ -29,20 +29,17 @@ function Template() {
 
 export const Default = Template.bind( {} );
 Default.storyName = 'Default';
-Default.parameters = {
-	features: [ 'audienceSegmentation' ],
-};
 Default.scenario = {
-	label: 'Components/AudienceSegmentationIntroductoryOverlayNotification',
+	label: 'Modules/Analytics4/Components/AudienceSegmentation/AudienceSegmentationIntroductoryOverlayNotification',
 };
 
 export default {
-	title: 'Components/AudienceSegmentationIntroductoryOverlayNotification',
+	title: 'Modules/Analytics4/Components/AudienceSegmentation/AudienceSegmentationIntroductoryOverlayNotification',
 	component: AudienceSegmentationIntroductoryOverlayNotification,
 	decorators: [
-		( Story, { parameters } ) => {
+		( Story ) => {
 			return (
-				<WithTestRegistry features={ parameters.features }>
+				<WithTestRegistry>
```

Comment:
While it doesn't hurt, I don't think we need the `<WithTestRegistry>` wrapper here anymore. The entire `decorators` property can go away. Let me know what you think, thanks!